lib/osf-components/addon/components/file-browser/template.hbs

```diff
@@ -1,163 +1,165 @@
+                        >
+                            <span local-class='SortedBy {{if this.isMobile 'Mobile'}}'>
+                                {{t (concat 'osf-components.file-browser.sort_by.' @manager.sort)}}
+                            </span>
+                            <FaIcon @icon='caret-down' />
+                        </dropdown.trigger>
+                    </label>
+                    <dropdown.content
+                        local-class='DropdownList'
+                        {{on 'click' dropdown.close}}
+                    >
+                        <Button
+                            @layout='fake-link'
+                            {{on 'click' (fn @manager.changeSort 'name')}}
+                        >
+                            {{t 'osf-components.file-browser.sort_by.name'}}
+                        </Button>
+                        <Button
+                            @layout='fake-link'
+                            {{on 'click' (fn  @manager.changeSort '-name')}}
+                        >
+                            {{t 'osf-components.file-browser.sort_by.-name'}}
+                        </Button>
+                        <Button
+                            @layout='fake-link'
+                            {{on 'click' (fn  @manager.changeSort 'modified')}}
+                        >
+                            {{t 'osf-components.file-browser.sort_by.modified'}}
+                        </Button>
+                        <Button
+                            @layout='fake-link'
+                            {{on 'click' (fn  @manager.changeSort '-modified')}}
+                        >
+                            {{t 'osf-components.file-browser.sort_by.-modified'}}
+                        </Button>
+                    </dropdown.content>
+                </ResponsiveDropdown>
             {{/if}}
-            <Button
-                data-test-file-help
-                data-analytics-name='Help'
-                aria-label={{t 'general.help'}}
-                @layout='fake-link'
-                {{on 'click' this.toggleHelpModal}}
-            >
-                <FaIcon @icon='question-circle' />
-            </Button>
-            <FileBrowser::HelpModal
-                @isOpen={{this.helpModalOpen}}
-                @onClose={{this.toggleHelpModal}}
-            />
-        {{/if}}
-    </div>
-</div>
-<InfiniteList
-    @itemList={{@manager.displayItems}}
-    @hasMore={{@manager.hasMore}}
-    @loadMore={{@manager.loadMore}}
-    @isLoading={{@manager.isLoading}}
-    local-class='FileList'
->
-    <:header>
-        {{#if @manager.parentFolder}}
-            <li local-class='GoToParentButton'>
+        </div>
+        <div local-class='OptionBar__right'>
+            {{#if (gt @manager.selectedFiles.length 0)}}
+                {{!-- TODO: actually make these do things --}}
+                <FaIcon
+                    @icon='check'
+                />
+                {{t 'general.move'}}
```

Comment:
I thought I caught this on Futa's PR, but while we're making things not show for registrations, could we have the move and delete buttons not show there?